package.json

```diff
@@ -109,8 +109,8 @@
     "hooks:pre-push": "node ./hooks/pre-push.js",
     "gen:meta": "node ./bootstrap/scripts/genMeta.js",
     "backend:updateDendronDeps": "node bootstrap/backend/updateDendronhqDeps.js",
-    "stash:unstaged": "git stash save -k 'pre-linting-stash'",
-    "stash:pop": "git stash && git stash pop stash@{1} && git read-tree stash && git stash drop",
+    "stash:unstaged": "./scripts/stash-unstaged.sh",
+    "stash:pop": "./scripts/stash-pop.sh",
```

Comment:
This was failing before, I think it's because Yarn uses its own shell parser to run these scripts, and wasn't handling the `{}` characters properly. Hoisted these scripts out to their own files to workaround it.

https://github.com/yarnpkg/berry/issues/2670